core/modules/image/image.field.inc

```diff
@@ -699,6 +704,40 @@ function image_field_formatter_settings_form($field, $instance, $view_mode, $for
   $element['image_load']['auto']['#description'] = t('Each browser will determine whether or not to lazily load the image.');
   $element['image_load']['eager']['#description'] = t('Load this image immediately, regardless of where it is located on the page.');
 
+  $element['image_svg_dimensions'] = array(
+    '#type' => 'item',
+    '#title' => t('Maximum SVG image dimensions'),
+    '#field_prefix' => '<div class="container-inline">',
+    '#field_suffix' => '</div>',
+    '#description' => t('The maximum allowed SVG image dimensions expressed as <code>WIDTH</code> x <code>HEIGHT</code>. If left blank, SVG images will be displayed at @svg_default_width x @svg_default_height pixels. Selecting an image style will override this setting.', array(
```

Comment:
TBH, I don't understand, what the purpose of this setting is.
We have a default value (constant) as fallback, which already was applied on svg's where dimension attributes were missing.
We already have image styles - which also ship with dimensions.
Why yet another layer? Could it be, that this is leftover from the markup display?

Can we entirely drop this? Otherwise it feels like reinventing the wheel (image styles).

Reply:
Do you still think we should drop this? It only applies for image fields and won't help with files uploaded at `admin/content/files` I think. So maybe drop it.

Reply:
I agree this can go, the size calculations are really too complex. I think this was part of the original code but I added the width constants, for some reason. Perhaps we could even remove the dimension constants as well, and just validate files that dont have set dimensions.

Reply:
Alright, let's get rid of the constants too, and add validation.

Reply:
Removed.

Reply:
@docwilmot actually the constants are in the drupal module too. https://git.drupalcode.org/project/svg_image/-/blob/7.x-1.x/svg_image.module?ref_type=heads